content/guides/_index.md

```diff
@@ -1,7 +1,5 @@
 ---
-title: 'Guides'
-deck: "TKTK"
-summary: 'TKTK'
-guide: guide
-redirectto: /resources
```

Comment:
Should resources point to `/guides` now?

Reply:
We don't need it since we now have a `list.html` for the guides.